robolectric/src/main/java/org/robolectric/RobolectricTestRunner.java

```diff
@@ -465,57 +472,16 @@ public void evaluate() throws Throwable {
     }
   }
 
-  private static class ManifestIdentifier {
-    private final FsFile manifestFile;
-    private final FsFile resDir;
-    private final FsFile assetDir;
-    private final String packageName;
-    private final List<FsFile> libraryDirs;
-
-    public ManifestIdentifier(FsFile manifestFile, FsFile resDir, FsFile assetDir, String packageName,
-        List<FsFile> libraryDirs) {
-      this.manifestFile = manifestFile;
-      this.resDir = resDir;
-      this.assetDir = assetDir;
-      this.packageName = packageName;
-      this.libraryDirs = libraryDirs != null ? libraryDirs : Collections.<FsFile>emptyList();
-    }
-
-    @Override
-    public boolean equals(Object o) {
-      if (this == o) return true;
-      if (o == null || getClass() != o.getClass()) return false;
-
-      ManifestIdentifier that = (ManifestIdentifier) o;
-
-      return assetDir.equals(that.assetDir)
-          && libraryDirs.equals(that.libraryDirs)
-          && manifestFile.equals(that.manifestFile)
-          && resDir.equals(that.resDir)
-          && ((packageName == null && that.packageName == null) || (packageName != null && packageName.equals(that.packageName)));
-    }
-
-    @Override
-    public int hashCode() {
-      int result = manifestFile.hashCode();
-      result = 31 * result + resDir.hashCode();
-      result = 31 * result + assetDir.hashCode();
-      result = 31 * result + (packageName == null ? 0 : packageName.hashCode());
-      result = 31 * result + libraryDirs.hashCode();
-      return result;
-    }
-  }
-
   // TODO: Instead of creating a dynamic proxy to set up a default return of the Config.class annotation,
   //   instead create a default constructor for Config.Implementation() which returns the default values of its fields.
   private static <A extends Annotation> A defaultsFor(Class<A> annotation) {
     return annotation.cast(
-        Proxy.newProxyInstance(annotation.getClassLoader(), new Class[] { annotation },
-            new InvocationHandler() {
-              public Object invoke(Object proxy, @NotNull Method method, Object[] args)
-                  throws Throwable {
-                return method.getDefaultValue();
-              }
-            }));
+            Proxy.newProxyInstance(annotation.getClassLoader(), new Class[] { annotation },
+                    new InvocationHandler() {
```

Comment:
What indentation is now used in Robolectric project for continuations? 4 or 6 spaces?